static/app/utils/analytics/replayAnalyticsEvents.tsx

```diff
@@ -0,0 +1,24 @@
+import type {LayoutKey} from 'sentry/utils/replays/hooks/useReplayLayout';
+
+export type ReplayEventParameters = {
+  'replay-details.layout-changed': {
```

Comment:
Should these be enums? They are used in multiple places. -- Will tsc catch if we change the event name when calling `trackAnalytics...`?

Reply:
The types here are really good actually. TSC matches the key with the args that it expects at the callsite to do the tracking.